cv/qc-cv.obo

```diff
@@ -2137,3 +2137,22 @@ property_value: has_optional_column: QC:4000249 ! PCA Dimension 3
 property_value: has_optional_column: QC:4000250 ! PCA Dimension 4
 property_value: has_optional_column: QC:4000251 ! PCA Dimension 5
 
+
+[Term]
+id: QC:4000268
+name: Number of peaks for at least 50% total ion current per spectrum
+def: "The number of peaks needed to cover at least 50% of the total ion current in individual spectra." [PSI:QC]
+comment: A simple metric to assess the quality of individual spectra using their peaks intensities compared to their total ion current.
+is_a: QC:4000006 ! table
+is_a: QC:4000010 ! ID free
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation QC:4000013 ! QC metric relation: single run
+property_value: has_column: QC:4000269 ! Universal Spectrum Identifier
+property_value: has_column: QC:4000115 ! Count
+
+[Term]
+id: QC:4000269
+name: Universal Spectrum Identifier
+def: "The column contains Universal Spectrum Identifiers to refer to individual spectra." [PSI:QC]
```

Comment:
mention "USI" somewhere? To make it searchable. E.g. imagine someone who sees "This is an USI, used for..." and has no clue what it means...